### PR TITLE
Added warnings to select Pytorch mobile tutorials directing users to ExecuTorch 

### PR DESCRIPTION
**Pull request by tstatler**
Addresses https://www.internalfb.com/tasks/my_tasks?t=200046828

Preview links for changed files:

- https://docs-preview.pytorch.org/pytorch/tutorials/3016/beginner/deeplabv3_on_android.html
- https://docs-preview.pytorch.org/pytorch/tutorials/3016/recipes/mobile_perf.html
- https://docs-preview.pytorch.org/pytorch/tutorials/3016/recipes/ptmobile_recipes_summary.html
- https://docs-preview.pytorch.org/pytorch/tutorials/3016/prototype/lite_interpreter.html (Note: this file was renamed so this URL now redirects to https://docs-preview.pytorch.org/pytorch/tutorials/3016/recipes/mobile_interpreter.html, which is added in this PR.)

## Description

This PR adds warnings to select Pytorch mobile tutorials directing users to ExecuTorch.

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

cc @svekars @brycebortree

### CODE REVIEW

**Review by svekars — APPROVED** [review]

**Comment by tstatler** [comment]
@ali-khosh @svekars FYI after adding the preview links I realized I had a left a `info` block instead of `warning` on https://docs-preview.pytorch.org/pytorch/tutorials/3016/beginner/deeplabv3_on_android.html . I merged a fix but will take a while for it to appear on the preview page.

**Review by ali-khosh — APPROVED** [review]
Reviewed changes to various pages having a warning note on top that says PT Mobile is deprecated.

File: prototype_source/lite_interpreter.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(Prototype) Introduce lite interpreter workflow in Android and iOS
+=======================
+
+This tutorial has been moved to https://pytorch.org/tutorials/recipes/mobile_interpreter.html
+
+It will redirect in 3 seconds.
+
+.. raw:: html
+
+   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/tutorials/recipes/mobile_interpreter.html'" />
```
Comment (svekars) [inline prototype_source/lite_interpreter.rst:10]: Maybe let's just make it instant. 
```suggestion
   <meta http-equiv="Refresh" content="0; url='https://pytorch.org/tutorials/recipes/mobile_interpreter.html'" />
```

File: prototype_source/lite_interpreter.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(Prototype) Introduce lite interpreter workflow in Android and iOS
+=======================
+
+This tutorial has been moved to https://pytorch.org/tutorials/recipes/mobile_interpreter.html
+
+It will redirect in 3 seconds.
```
Comment (svekars) [inline prototype_source/lite_interpreter.rst:6]:
```suggestion
```

**Pull request merged**